helm/files/bootstrap.py

```diff
@@ -1,355 +0,0 @@
-#!/usr/bin/env python
```

Comment:
are we expecting customers to use the bootstrap.py in the `privatedns` repo?

Reply:
I didn't want to be prescriptive about the bootstrapping process, so I'm not sure. I think having it within the helm chart here + having the bootstrap UI is confusing so I removed the scripting part here